src/diffusers/utils/peft_utils.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+PEFT utilities: Utilities related to peft library
+"""
+from .import_utils import is_torch_available
+
+
+if is_torch_available():
+    import torch
+
+
+def recurse_remove_peft_layers(model):
```

Comment:
I wonder if we can somehow use `unload` from PEFT? The logic seems pretty similar. Maybe it would work with some small tweaks?

Reply:
Yeah, it would require some tweaks on PEFT side. Maybe we can add those later to avoid code duplication, it should be fine as is for this PR.

Reply:
yes, we could think of exposing a public method that will be leveraged by `unload` method in `LoraModel` and use that later in diffusers